languages/csharp/Bitwarden.Sdk/README.md

```diff
@@ -0,0 +1,58 @@
+# Bitwarden .NET SDK
```

Comment:
We should be explicit that this is only the secrets manager SDK.
```suggestion
# Bitwarden Secrets Manager SDK
```

Reply:
done